app/controllers/bookings_controller.rb

```diff
@@ -104,6 +105,14 @@ def ticket_params
     params.require(:tickets_quantity)
   end
 
+  def source_params
+    params.permit(:event_source, :other)
```

Comment:
What's the need for `params.permit...` here, given that the returned is later manipulated to give the required `params[:event_source]` ?